JSDemos/Demos/Toolbar/Multiline/jQuery/index.js

```diff
@@ -0,0 +1,376 @@
+const toolbarDataDefault = [
+  {
+    locateInMenu: 'never',
```

Comment:
LocateInMenu option has 'never' as default value. I think we may remove it's from all configuration where it uses or check that behavior is not changed.

Reply:
https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#locateInMenu

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/289ebcb27f25b17b6c07714babdff8ec200f788c